project/ProjectPlugin.scala

```diff
@@ -1,48 +1,114 @@
-      addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"             % "0.4.0"),
-      addSbtPlugin("com.github.gseitz"  % "sbt-release"            % "1.0.4"),
-      addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"           % "1.1"),
-      addSbtPlugin("com.jsuereth"       % "sbt-pgp"                % "1.0.1"),
-      addSbtPlugin("com.typesafe.sbt"   % "sbt-ghpages"            % "0.6.0"),
-      addSbtPlugin("com.typesafe.sbt"   % "sbt-site"               % "1.2.0"),
-      addSbtPlugin("org.tpolecat"       % "tut-plugin"             % "0.5.4"),
-      addSbtPlugin("pl.project13.scala" % "sbt-jmh"                % "0.2.24"),
-      addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin" % "0.8.0"),
-      addSbtPlugin("org.scoverage"      % "sbt-scoverage"          % "1.5.0"),
-      addSbtPlugin("org.scala-js"       % "sbt-scalajs"            % "0.6.18"),
-      addSbtPlugin("de.heikoseeberger"  % "sbt-header"             % "1.8.0"),
+      addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.9.3" commonExcludes),
+      addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"             % "0.4.1"),
+      addSbtPlugin("com.github.gseitz"  % "sbt-release"            % "1.0.6"),
+      addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"           % "2.0"),
+      addSbtPlugin("com.jsuereth"       % "sbt-pgp"                % "1.1.0-M1"),
+      addSbtPlugin("com.typesafe.sbt"   % "sbt-ghpages"            % "0.6.2"),
+      addSbtPlugin("com.typesafe.sbt"   % "sbt-site"               % "1.3.0"),
+      addSbtPlugin("pl.project13.scala" % "sbt-jmh"                % "0.2.27"),
+      addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin" % "1.0.0"),
+      addSbtPlugin("org.scoverage"      % "sbt-scoverage"          % "1.5.1"),
+      addSbtPlugin("org.scala-js"       % "sbt-scalajs"            % "0.6.19"),
+      addSbtPlugin("de.heikoseeberger"  % "sbt-header"             % "3.0.1"),
       addSbtPlugin("com.eed3si9n"       % "sbt-buildinfo"          % "0.7.0"),
-      addSbtPlugin("com.geirsson"       % "sbt-scalafmt"           % "0.6.8"),
-      addSbtPlugin("io.get-coursier"    % "sbt-coursier"           % "1.0.0-RC9"),
-      addSbtPlugin("com.47deg"          % "sbt-dependencies"       % "0.1.1"),
-      addSbtPlugin("com.47deg"          % "sbt-microsites"         % "0.6.1" exclude("org.tpolecat", "tut-plugin"))
-    ) ++
-      ScriptedPlugin.scriptedSettings ++ Seq(
-      scriptedDependencies := (compile in Test) map { _ =>
-        Unit
-      },
+      // addSbtPlugin("com.lucidchart"     % "sbt-scalafmt"  % "1.10"),
+      // addSbtPlugin("com.geirsson"       % "sbt-scalafmt"  % "1.2.0"),
+      libraryDependencies ++= {
+        val sbtVersionValue       = (sbtVersion in pluginCrossBuild).value
+        val sbtBinaryVersionValue = (sbtBinaryVersion in pluginCrossBuild).value
+
+        val scalaBinaryVersionValue = (scalaBinaryVersion in update).value
+
+        sbtVersionValue match {
```

Comment:
Let's use this pattern match to only extract the tut and scalafmt deps, leaving just one `Seq` of settings at the end. What do you think?

Reply:
Do you mean use the pattern match only and return `("0.5.3", "0.6.8")` or `("0.6.8", "1.2.0")`? DRY up the repeated group id and artifact id? If so, makes sense.

Reply:
@juanpedromoreno bump

Reply:
Yes, that's what I meant ;)